packages/headless/src/api/search/search/product-recommendation.ts

```diff
@@ -2,69 +2,89 @@ export interface ProductRecommendation {
   /**
    * The SKU of the product.
    */
-  sku: string;
+  permanentid: string;
+  /**
+   * A direct link to the product in URL format.
+   */
+  clickUri: string;
```

Comment:
Still not sure about `permanentid` and `clickUri`, I am thinking about keeping `sku` and `link` since they are clearer to me

Reply:
You may want both `permanentid` and `sku`. `permanentid` can be a sku or a productid. sku are usually set on variants, and not on products. So having them on Product-recommendations may be confusing.